lib/Sema/TypeCheckDeclPrimary.cpp

```diff
@@ -293,6 +293,20 @@ static void checkInheritanceClause(
         }
         continue;
       }
+
+      // Noncopyable types cannot have a raw type until there is support for
+      // generics, since the raw type here is only useful if we'll generate
+      // a conformance to RawRepresentable, which is currently disabled.
+      if (enumDecl->isMoveOnly()) {
+        // TODO: getRemovalRange is not yet aware of ~Copyable entries so it
+        // will accidentally delete commas or colons that are needed.
+        diags.diagnose(inherited.getSourceRange().Start,
+                       diag::enum_raw_type_nonconforming_and_noncopyable,
+                       enumDecl->getDeclaredInterfaceType(), inheritedTy)
+             .highlight(inherited.getSourceRange());
+
+        // Fall through to still record this raw type.
```

Comment:
This comment is out of date (I know it was copy and pasted from below). We don't record anything in checkInheritanceClause(); it is just there to emit diagnostics.

Reply:
ah okay. will remove!